app/src/org/commcare/models/legacy/LegacyInstallUtils.java

```diff
@@ -231,38 +225,18 @@ public static void checkForLegacyInstall(Context c, SqlStorage<ApplicationRecord
         Logger.log(LogTypes.TYPE_MAINTENANCE, "Legacy| Files moved. Updating Handles");
 
         //We also need to tell the XForm Provider that any/all of its forms have been moved
-
-        ArrayList<Pair<Uri, String>> toReplace = new ArrayList<>();

```

Comment:
**Important Note**: -This code will no longer work after this change. This legacy installer should continue assuming that a form provider exists (using hard-coded strings or whatever else is retained)  **or** we should kill the legacy updater code.

Reply:
Yeah. Didn't realize this block of code runs before DB update. We can defnitely keep the Content Provider code if it's worth it. What all versions of CommCare does legacy actually represents ?

Reply:
Confirmed over slack, but the Legacy db represents CommCare 2.3.0 and below, so we can move forward with the assumption that we are removing legacy migration code.